python/scripts/populate-minion.py

```diff
@@ -35,7 +35,7 @@
             pipe.MotionCorrection().populate(next_scans, reserve_jobs=True, suppress_errors=True)
             pipe.SummaryImages().populate(next_scans, reserve_jobs=True, suppress_errors=True)
             pipe.Segmentation().populate(next_scans, reserve_jobs=True, suppress_errors=True)
-            pipe.Fluorescence().populate(next_scans, reserve_jobs=True, suppress_errors=True)
+            # pipe.Fluorescence().populate(next_scans, reserve_jobs=True, suppress_errors=True)
```

Comment:
why is this commented out?

Reply:
I commented this out because we are populating `Flourescence()` when we populate  `pipe.Segmentation()`. So we are calling `Flouresence` for no reason

Reply:
Fluorescence will still need to be computed for Manual masks

Reply:
fixed!